package.json

```diff
@@ -58,6 +58,7 @@
     "jsdom-global": "^3.0.2",
     "keycode": "^2.2.0",
     "lodash": "^4.17.19",
+    "merge": "^2.1.1",
```

Comment:
I checked that the `merge` package is only used by `sass-lint` which is already upgraded to maximum. In this case pinning this dependency would just resolve dependabot alert but not actually do anything else since `sass-lint` will still use `merge(1.2.0)` inside.

Instead of pinning we should go for dependency resolution and check that it doesn't breaking any existing thing You can take a look at this link for reference https://classic.yarnpkg.com/en/docs/selective-version-resolutions.

Reply:
Do you mean merge should be in `resolutions ` ?

```
"resolutions": {
    "merge": "^2.1.1"
  },
```